README.md

```diff
@@ -1,54 +1,149 @@
 # BizHawk
 
-BizHawk is a multi-system emulator written in C#. BizHawk provides nice features for casual gamers such as fullscreen, rewind, and joypad support in addition to rerecording and debugging tools for all system cores.
+[![GitHub latest release](https://img.shields.io/github/release/TASVideos/BizHawk.svg?logo=github&style=flat)](https://github.com/TASVideos/BizHawk/releases/latest) ![Unique systems emulated | 22](https://img.shields.io/badge/Unique_systems_emulated-22-darkgreen.svg?style=flat)
 
-[Release Notes](http://tasvideos.org/Bizhawk/ReleaseHistory.html)
+A multi-system emulator written in C#. As well as quality-of-life features for casual players, it also has recording/playback and debugging tools, making it the first choice for TASers (Tool-Assisted Speedrunners).
 
-[Here](http://tasvideos.org/Bizhawk/Features.html) is a list of features offered by bizhawk.
+## Core feature matrix
 
-## Download Binaries
+Most games for every system in the table will run perfectly, and every system supports fast-forward, frame advance, and rewind.
 
-Windows users, don't forget to run the [prereq installer](http://github.com/TASVideos/BizHawk-Prereqs/releases) first!
+System | Play+FF/ADV/RW | Debugging | (Input) Recording
+--:|:-:|:-:|:-:
+Apple II | ✓ | ? | ?
+Atari 2600 | ✓ | ? | ?
+Atari 7800 | ✓ | ? | ?
+Atari Lynx | ✓ | ? | ?
+ColecoVision | ✓ | ? | ?
+Commodore 64 | ✓ | ? | ?
+Game Boy / GBC | ✓ | ? | ?
+GBA | ✓ | ? | ?
+Genesis | ✓ | ? | ?
+IntelliVision | ✓ | ? | ?
+Master System | ✓ | ? | ?
+N64 | ✓ | ? | ?
+Neo Geo Pocket | ✓ | ? | ?
+NES | ✓ | ? | ?
+PSX | ✓ | ? | ?
+Saturn | ✓ | ? | ?
+SNES | ✓ | ? | ?
+TI-83 | ✓ | ? | ?
+TurboGrafx-16 | ✓ | ? | ?
+Virtual Boy | ✓ | ? | ?
+WonderSwan | ✓ | ? | ?
+ZX Spectrum | ✓ | ? | ?
 
-Release binaries can be found on [on github](http://github.com/TASVideos/BizHawk/releases)
+EmuHawk's frontend also has a comprehensive input mapper, with keyboard and controller support (even simultaneously), hotkeys, and turbo/rapid fire.
 
-[Developer build](https://ci.appveyor.com/project/zeromus/bizhawk-udexo/build/artifacts) of the most recent commit
+In addition to input recording (with TAStudio), you can also record the emulated system's audio and video, either of your own gameplay or of a prerecorded movie (TAS). See the *Usage (advanced)* section below for more info about the many debugging and recording features.
 
-Never mix different versions of BizHawk. Keep each version in its own folder.
+Most of the frontend can be controlled with Lua scripts (only on Windows for now). These scripts can draw overlays for casual play or for RTA speedruns, change the running game to add complex cheats, assist TASers and romhackers in debugging a game, and more.
```

Comment:
Avoid mentioning cheats for now. They occasionally work, but no one cares about them and most of them are completely broken. We don't want people take them seriously and complain every day.

Reply:
It says Lua can be used to cheat e.g. randomisers, is that not clear?

Reply:
Hmm, ways to cheat are: writing to memory with Lua, changing and freezing memory in Hex Editor, freezing in ram watch/search, don't remember if they both have this, also if they can edit values. Game genie and gameshark are half-assed.

Reply:
this prose was removed